pages/HottestSongs.tsx

```diff
@@ -190,7 +195,8 @@ const HottestSongs: React.FC = () => {
           imageUrl={ipfsToHttps(selectedSong.advNfts[0]?.assetHash ?? "")}
         />
       )}
-      <SongList playSong={handlePlaySong} />
+      {/* <SongList playSong={handlePlaySong} /> */}
```

Comment:
Remove this line

Reply:
Oh sorry! forgot to remove it after testing the memoized version